packages/paste-theme-tokens/src/default/index.ts

```diff
@@ -35,6 +36,11 @@ export const DefaultTheme = {
   maxHeights: sizings,
   minHeights: sizings,
   sizes: sizings,
+  iconSizes: {
+    sizeIcon10: sizings.sizeIcon10,
+    sizeIcon20: sizings.sizeIcon20,
+    sizeIcon30: sizings.sizeIcon30,
+  },
```

Comment:
I tried automating this with an `Object.keys().filter().forEach()` but the resulting typings weren't as clean.